screens/Answers.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from 'react';
+import { View, Text, Image, TouchableOpacity } from 'react-native';
+import styled from 'styled-components/native';
+
+import { YES_NO_API } from '../reuseables/urls'
+
+// This is the main container for this screen
+const NotificationsContainer = styled.View`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  background-color: black;
+`;
+
+const ImageGiph = styled.Image`
+  width: 350px;
+  height: 350px;
+`;
+
+const AnswerText = styled.Text`
+  color: yellow;
+  font-family: Courier;
+  font-weight: bold;
+  font-size: 35px;
+  margin: 20px 0 20px 0;
+`;
+
+const BackButton = styled.TouchableOpacity`
+  position: absolute;
```

Comment:
you have an absolute position set for your button, so if I have a taller screen it will be higher than on the smaller screens. if you remove it I think it should layout your button under the text.